.rubocop.yml

```diff
@@ -12,3 +12,9 @@ Metrics/BlockLength:
 
 Style/Documentation:
   Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/MethodLength:
+  Enabled: false
```

Comment:
These metrics are useful to achieve a good code structure, you can increase the default limits if you will but please keep them enabled.